plugins/in_tail/win32/stat.c

```diff
@@ -44,6 +46,18 @@ static int get_mode(unsigned int attr)
     return WIN32_S_IFREG;
 }
 
+static int64_t filetime_to_epoch(FILETIME ft)
+{
+    int64_t ldap;
```

Comment:
Please use a [ULARGE_INTEGER](https://learn.microsoft.com/en-us/windows/win32/api/winnt/ns-winnt-ularge_integer-r1) to store the QWORD instead of that custom cast.